man/parser_standard_options.py

```diff
@@ -351,48 +360,51 @@ def _repr_html_(self):
         "--html_params",
         default="border=1",
         type=str,
-        dest="htmlparmas",
+        dest="htmlparams",
```

Comment:
Would using underscore be a better standard here helping to avoid misspellings?

Reply:
I don't think so, code spelling checking (cspell) in my IDE was already pointing it out.

The underscore would be in between html and params, and it was params that had a typo.

Reply:
I didn't see problems with the other argument destinations by the way

Reply:
@wenzeslaus Can you check back what's left on this? It's been a while